src/components/GlobalCheckClaimStatus/index.tsx

```diff
@@ -29,33 +32,55 @@ const GlobalCheckClaimStatus: React.FC<GlobalCheckClaimStatusProps> = (props) =>
  */
 const GlobalCheckClaim: React.FC<GlobalCheckClaimStatusProps> = ({ excludeLocations }) => {
   const hasDisplayedModal = useRef(false)
-  const [canClaimAnniversaryPoints, setCanClaimAnniversaryPoints] = useState(false)
+  const [cid, setCid] = useState(null)
+  const [canClaimNFT, setCanClaimNFT] = useState(false)
+  const [nftBalance, setNftBalance] = useState(999) // default high to avoid flashing modal
+  const galaxyNFTContract = useERC721('0x4344cDD59780CC7B8A47Cd903424F91f41A3a366')
   const { account } = useWeb3React()
   const { pathname } = useRouter()
-  const [onPresentAnniversaryModal] = useModal(<AnniversaryAchievementModal />)
+  const [onPresentModal] = useModal(<GalaxyNFTClaimModal cid={cid} />, false, true, 'galaxyNFTClaimModal')
```

Comment:
Close on overlay click cannot be disabled right now, there is a fix for that https://github.com/pancakeswap/pancake-frontend/pull/3814

Reply:
yeah its ok, I just need to put something in there to set up updating modal :)